package.json

```diff
@@ -16,7 +16,7 @@
   },
   "homepage": "https://github.com/oppia/oppia#readme",
   "devDependencies": {
-    "@lhci/cli": "^0.6.0",
+    "@lhci/cli": "^0.7.0",
```

Comment:
@vojtechjelinek, I think the lighthouse test failure is due to the change in version.